diff --git a/app/views/issues/show.html.erb b/app/views/issues/show.html.erb
index 6c0018f..67e8f4e 100644
--- a/app/views/issues/show.html.erb
+++ b/app/views/issues/show.html.erb
@@ -122,6 +122,8 @@ end %>
 
 <%= render partial: 'action_menu_edit' if User.current.wants_comments_in_reverse_order? %>
 
+<%= call_hook(:view_issues_show_after_details, :issue => @issue) %>
+
 <% if @changesets.present? %>
 <div id="issue-changesets">
 <h3><%=l(:label_associated_revisions)%></h3>
